eda/setup.py

```diff
@@ -24,7 +24,7 @@
     'scipy',
     'scikit-learn',
     'pandas',
-    'matplotlib',
+    'matplotlib>3.3',
```

Comment:
Do we want to upper cap it?

Reply:
We don''t have a cap anywhere else. But I need the bottom cap because old versions break functionality (some features are missing).